participants/anthony.json

```diff
@@ -0,0 +1,36 @@
+{
+    // required
+    "name": "Anthony Craig",
+    // optional
+    "company": "your company name",
+    // required
+    "when": {
+        "friday": true,
+        "saturday": false
+    },
+    // required
+    "iCanTakeNotesDuringSessions": false,
+    // required - at least one
+    "tags": ["ReScript"],
+    // optional
+    "vegan": false,
+    // optional
+    "vegetarian": true,
+    // optional
+    "allergies": ["Gluten", "Peanuts"],
```

Comment:
Are you really allergic to both our sample allergy options?

Reply:
I am not allergic to both